source/installation/vm/tangobox.rst

```diff
@@ -0,0 +1,367 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Prior to use `HDB/TDB` (:guilabel:`Mambo`) or `SNAP` (:guilabel:`Bensikin`) you need to make sure that
+the  **tangobox-archiving** container and related device servers are running:
+
+* Call :command:`docker start tangobox-archive` on a terminal.
+* Start :program:`Astor` and check if the `tangobox-archive` node is green.
+
+Then, you may start :program:`Mambo` or :program:`Bensikin` by clicking icons on the desktop.
+
+.. figure:: tangobox/mambo.png
+
+   Screen of running Mambo
+
+   Please take note of a green bulb of the `tangobox-archive` node in the Astor window.
+
+
+.. _hdbpp:
+
+HDB++ Archiving
+~~~~~~~~~~~~~~~
+
+To use `HDB++` and its tools ( :guilabel:`HDB Configurator` and :guilabel:`HDB Viewer`) please make sure that
+the  **tangobox-hdbpp** container and related device servers are running:
+
+* Call :command:`docker start tangobox-hdbpp` on a terminal.
+* Start :program:`Astor` and check if the `tangobox-hdbpp` node is green.
+
+Then, you may start :program:`HDB Configurator` or :program:`HDB Viewer` by clicking icons on the desktop.
+
+.. _e-giga:
+
+E-giga
+~~~~~~
+
+:program:`E-giga` is a web application for archiving data visualization (HDB/TDB and HDB++). The TangoBox deployment
+uses HDB/TDB.
+
+.. figure:: tangobox/e-giga.png
+
+   E-giga in a web browser window
+
+To use e-giga following conditions must be fulfilled:
+
+* `tangobox-archive` and `tangobox-web` containers must be started and archiving device servers must be running
+* use i.e. **Mambo** to enable data archiving for HDB database. It is required. If you do not see any attributes in
+  `E-giga` it is probably due to archiving being disabled. Check with :program:`Astor` if the tangobox-archive LED
+  is green and with :program:`Mambo` if there are any attributes configured to be archivied.
+
+To open browser with :program:`E-giga` click on the relate desktop icon.
+
+.. note::
+
+   Please keep in mind that you should not rebuild **tangobox-web** image because its configuration is not included in Dockerfile
+   (it requires in-container config).
+
+.. _webapp:
+
+Tango WebApp
+~~~~~~~~~~~~
+
+Tango may be available through a web browser. `Tango WebApp` is a general purpose Tango web application. You may try it
+on the TangoBox.
+
+.. figure:: tangobox/webapp-2.png
+
+   A screenshot of Tango WebApp in a browser
+
+To play with :program:`Tango WebApp` make sure that the 'tangobox-web` container is running
+(use :command:`docker start tangobox-web` to start it from a terminal). Then, you may open a browser with a related
+desktop icon. Use username `tango-cs` and  password `tango` to log-in.
+
+
+.. _rest_api:
+
+REST API
+~~~~~~~~
+
+Tango Controls specifies REST API interface and provides its reference implementation. For details see
+:ref:`REST API documentation <tango_rest_api>`
+
+The **TangoBox** comes with REST API installed. The `tangobox-web` container must be started to play with it. Invoke
```

Comment:
Rest server has been moved to the host machine. It is no longer included in tangobox-web container.